lib/ui/platform_dispatcher.dart

```diff
@@ -758,6 +758,14 @@ class PlatformDispatcher {
   ///
   /// In either case, this function disposes the given update, which means the
   /// semantics update cannot be used further.
+  @Deprecated('''
+  In a multi-view world, the platform dispatcher can no longer provide apis
```

Comment:
nit: indent these lines by two so they are indented more than the `@Deprecated` line.

Reply:
(here and for all other `@Deprecarted` instances below as well.)